### ISSUE TITLE
Exposing a Minecraft Bedrock Server

### ISSUE THREAD

**Issue by BaksiLi** [jpillora/chisel#456]
I deployed Chisel to expose my Minecraft Bedrock server (UDP), which runs from my home, and was successfully able to bind it to a domain. It is more stable than the previous tunnel I built with [FRP](https://github.com/fatedier/frp).

I wondered if it would benefit for chisel if I **documented this process and turned it into a short tutorial**, something similar to example/reverse-tunneling-authenticated.md. Is this use case too specific to be an example?

**Comment by Servostar** [comment]
I think it would be favorable to have a tutorial (maybe even a . Chisel is comparatively simple in terms of setup (client, server and a few cmd args), but especially inexperienced users could make great usage of this. Recently a friend of mine had some trouble to properly understand and setup chisel for his homelab.

And I do not believe that it is too specific, since I mainly use it for reverse tunneling too. Additionally it shouldn't be  a big deal to add a few tutorials for standard port forwarding or socks5.

**Comment by juanluisbaptiste** [comment]
Yes please! I'm looking to do the same with another game server.
